tensorboard/main.py

```diff
@@ -231,6 +232,7 @@ def main(unused_argv=None):
         graphs_plugin.GraphsPlugin,
         distributions_plugin.DistributionsPlugin,
         histograms_plugin.HistogramsPlugin,
+        profile_plugin.ProfilePlugin,
```

Comment:
nit: these plugins are arranged in order of tab appearance. mind moving to the end for consistency?